avm/res/app-configuration/configuration-store/main.bicep

```diff
@@ -301,7 +301,10 @@ resource configurationStore_roleAssignments 'Microsoft.Authorization/roleAssignm
 module configurationStore_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.7.1' = [
   for (privateEndpoint, index) in (privateEndpoints ?? []): {
     name: '${uniqueString(deployment().name, location)}-configurationStore-PrivateEndpoint-${index}'
-    scope: resourceGroup(privateEndpoint.?resourceGroupName ?? '')
+    scope: resourceGroup(
```

Comment:
An overdue update - but please note that because the type import was not updated, the `resourceGroupResourceId` parameter will not be available via the parameter. Please also update the private-endpoint type import to the latest available version.